.github/ISSUE_TEMPLATE/bug.yaml

```
@@ -0,0 +1,23 @@
+name: 🐞 Bug
+description: Found a bug? We are sorry about that! Let us know! 🐛
+title: "[BUG] "
+labels: [bug, Needs Triage]
+body:
+  - type: textarea
+    attributes:
+      label: Observed behavior
+      description: What are you trying to do? Describe what you think went wrong during this.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: A concise description of what you expected to happen.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: Describe as best you can the problem. Please provide us scenario file, logs or anything you have that can help us to understand.
```

```suggestion
      description: Describe as best you can the problem. Please provide us a scenario file, logs or anything you have that can help us to understand.
```

Possibly also mention versions?